ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -1510,9 +1510,8 @@ MiFindInitializationCode(OUT PVOID *StartVa,
             /* Assume failure */
             InitFound = FALSE;
 
-            /* Is this the INIT section or a discardable section? */
-            if ((strncmp((PCCH)Section->Name, "INIT", 5) == 0) ||
-                ((Section->Characteristics & IMAGE_SCN_MEM_DISCARDABLE)))
+            /* Is this a discardable section? */
+            if (Section->Characteristics & IMAGE_SCN_MEM_DISCARDABLE)
```

Comment:
The removal of the check for "INIT" section breaks Windows compatibility (you can check that by testing whether a section named `"INIT"` gets indeed discarded, even if it doesn't have the `IMAGE_SCN_MEM_DISCARDABLE` flag set).

Reply:
Also, there is extra treatment for other sections with special names, as explained in
https://community.osr.com/discussion/comment/162734/#Comment_162734

Reply:
Right. Commit reverted.